camptix.php

```diff
@@ -5480,7 +5503,8 @@ function form_attendee_info() {
 							$questions = $this->get_sorted_questions( $ticket->ID );
 						?>
 						<input type="hidden" name="tix_attendee_info[<?php echo esc_attr( $i ); ?>][ticket_id]" value="<?php echo intval( $ticket->ID ); ?>" />
-						<table class="tix_tickets_table tix-attendee-form">
+						<table class="tix_tickets_table tix-attendee-form <?php echo ( isset($this->options['select2_enabled']) && (bool) ( $this->options['select2_enabled'] ) ) ?
```

Comment:
good to add it here, but need to use on currency and summarize fields in wp-admin

Reply:
There is a separate patch for currency, since its not a part of camptix. Or am I confusing with a different currency dropdown ?

Reply:
Yeah. CampTix has a currency dropdown on the `Tickets > Setup > General` screen. The Summarize one is under `Tickets > Tools`.

Reply:
Ah ok. Added.